packages/e2e/example/test/example_e2e_test.dart

```diff
@@ -5,6 +5,8 @@
 // gestures. You can also use WidgetTester to find child widgets in the widget
 // tree, read text, and verify that the values of widget properties are correct.
 
+@TestOn('!vm')
```

Comment:
I think something about this line is getting it try and build for macOS targets.

```
Detected changes in the following 1 package(s):
e2e

running build-examples --macos --no-ipa

BUILDING macos for e2e/example
Running "flutter pub get" in example...                             7.1s
Downloading darwin-x64 tools...                                     2.0s
Downloading darwin-x64-profile tools...                             1.0s
Downloading darwin-x64-profile tools...                             0.6s
Downloading darwin-x64-release tools...                             1.0s
Downloading darwin-x64-release tools...                             0.6s
No macOS desktop project configured. See https://flutter.dev/desktop#add-desktop-support-to-an-existing-flutter-project to learn about adding macOS support to a project.
```